docs/design/proposals/encryption-with-vault-tokens.md

```diff
@@ -233,20 +233,19 @@ stringData:
     KNSC2DVVXcCgkqcCgAwIBAgIwewrvx...
 ```
 
-Its also possible that a user can create a single secret for the certificates
-and update the configuration to fetch certificates from a secret.
+Its also possible that a user can create a single Secret that contains both
```

Comment:
`Its` -> `It is` (preferred) or `It's`

Maybe better:

> It is also possible for a user to create ...

and end the sentence with a `.`?

Reply:
@nixpanic done . PTAL.